src/home/rendering/render-github-login-button.ts

```diff
@@ -18,7 +18,7 @@ export async function checkSupabaseSession() {
   return session;
 }
 
-async function gitHubLoginButtonHandler() {
+export async function gitHubLoginButtonHandler() {
```

Comment:
Any reason for this export? I do not see it called anywhere in the content of this pull request.

Reply:
Before the auth setup was changed from relying on `localStorage` I was using this to force a login to remove the burden from the user in having to manually refresh, as it's no longer needed I'll remove the export